test/dynamo/test_utils.py

```diff
@@ -326,6 +329,83 @@ def test_dynamo_timed(self, mock_time, mock_time_ns):
         )
 
 
+class TestInductorConfigParsingForLogging(TestCase):
+    """
+    Test for parsing inductor config for logging in CompilationMetrics.
+    """
+
+    class TestObject:
+        def __init__(self, a, b):
+            self.a = a
+            self.b = b
+
+    """
+    Sanity check if the actual inductor config is parsaed correctly
+    """
+
+    def test_inductor_config_jsonify(self):
+        try:
+            inductor_config_json = utils._scrubbed_inductor_config_for_logging()
+        except Exception as e:
+            self.fail(
```

Comment:
I don't think you need to explicitly catch the exception. Test will fail if there's an exception